src/img/index.js

```diff
@@ -44,6 +44,9 @@ import asparagusGrowing from './items/asparagus-growing.png'
 import asparagusSeed from './items/asparagus-seed.png'
 import carrot from './items/carrot.png'
 import carrotGrowing from './items/carrot-growing.png'
```

Comment:
Do we need this old version of `carrotGrowing` anymore? If not, can we remove it?

Reply:
good catch, can indeed remove this